charts/ssi-credential-issuer/templates/job-issuer-migrations.yaml

```diff
@@ -70,14 +70,59 @@ spec:
           - name: "SEEDING__TESTDATAENVIRONMENTS__0"
             value: "{{ .Values.migrations.seeding.testDataEnvironments }}"
```

Comment:
should be removed because consortia files are getting removed

Reply:
removed